vizro-core/src/vizro/_themes/create_chart_templates.py

```diff
@@ -0,0 +1,137 @@
+import argparse
+import json
+import re
+import sys
+from pathlib import Path
+from typing import Optional
+
+from plotly import graph_objects as go
+from plotly.utils import PlotlyJSONEncoder
+
+from vizro._themes._color_values import COLORS
+from vizro._themes.common_values import create_template_common
+
+THEMES_FOLDER = Path(__file__).resolve().parent
+CSS_FILE = THEMES_FOLDER.parent / "static" / "css" / "vizro-bootstrap.min.css"
+VARIABLES = ["--bs-primary", "--bs-secondary", "--bs-tertiary-color", "--bs-border-color", "--bs-body-bg"]
+
+
+def _extract_last_two_occurrences(variable: str, content: str) -> tuple[Optional[str], Optional[str]]:
+    """Extracts the last two occurrences of a variable from the CSS content.
+
+    Within the `vizro-bootstrap.min.css` file, variables appear multiple times: initially from the default Bootstrap
+    values, followed by the dark theme, and lastly the light theme. We are interested in the final two occurrences,
+    as these represent the values for our dark and light themes.
+    """
+    pattern = re.compile(rf"{variable}:\s*([^;]+);")
+    matches = pattern.findall(content)
+    if len(matches) >= 2:  # noqa: PLR2004
+        return matches[-2].strip(), matches[-1].strip()
+
+    return None, None
+
+
+def extract_bs_variables_from_css_file(
+    variables: list[str], css_file_path: Path
+) -> tuple[dict[str, Optional[str]], dict[str, Optional[str]]]:
+    """Extract the last two occurrences for each variable in the CSS file."""
+    extracted_dark = {}
+    extracted_light = {}
+
+    with open(css_file_path) as css_file:
+        css_content = css_file.read()
```

Comment:
```suggestion
        css_content = css_file.read_text()
```
Pathlib makes this sort of thing much easier.